pkg/configs/db/postgres/postgres.go

```diff
@@ -134,6 +134,13 @@ func (d DB) findConfigs(filter squirrel.Sqlizer) (map[string]userconfig.View, er
 		cfg.DeletedAt = deletedAt.Time
 		cfgs[userID] = cfg
 	}
+
+	// Check for any errors encountered.
+	err = rows.Err()
+	if err != nil {
+		return nil, err
+	}
+
 	return cfgs, nil
```

Comment:
```suggestion
	return cfgs, rows.Err()
```

Can we simplify it to this?

Reply:
I guess the only issue to change to this is that the method will end up returning a partial result and an error in the case that one of the rows returns an error when Next is called. I think that most examples of this I see would return a nil pointer and an error in this case.

Depends on how you want the API to behave. Let me know your thoughts, I can make the change if required.

Depends on whether this

Reply:
> I guess the only issue to change to this is that the method will end up returning a partial result and an error in the case that one of the rows returns an error when Next is called. 

That's fine, code calling these two functions ignores partial result when error is returned. If you have a strong preference for returning `nil`, that's fine as well.

Reply:
Was going to have a look at this tonight, but looks like I'm too late! :-)